developer-docs-site/docs/concepts/staking.md

```diff
@@ -39,13 +35,22 @@ Below is a summary flow diagram of how staking on the Aptos blockchain works. Th
 
 ### How a custodian can stake on Aptos
 
-The Aptos staking module defines a capability that represents ownership. See [https://github.com/aptos-labs/aptos-core/blob/0daade4f734d1ba29a896b00d7ddde2249e87970/aptos-move/framework/aptos-framework/sources/configs/stake.move#L85](https://github.com/aptos-labs/aptos-core/blob/0daade4f734d1ba29a896b00d7ddde2249e87970/aptos-move/framework/aptos-framework/sources/configs/stake.move#L85).
+The Aptos staking module defines a capability that represents ownership. 
+
+:::tip Ownership
+See [https://github.com/aptos-labs/aptos-core/blob/0daade4f734d1ba29a896b00d7ddde2249e87970/aptos-move/framework/aptos-framework/sources/configs/stake.move#L85](https://github.com/aptos-labs/aptos-core/blob/0daade4f734d1ba29a896b00d7ddde2249e87970/aptos-move/framework/aptos-framework/sources/configs/stake.move#L85).
+:::
 
-This `OwnerCapability` resource can be used to control the stake pool. Three personas: the owner, the operator and the voter, are supported. Using this owner-operator-voter model, a custodian can assume the owner persona and stake on the Aptos blockchain, and participate in the Aptos governance. This model allows delegations and staking services to be built as the owner can provide funds to the validator and the voter personas.
+The `OwnerCapability` resource can be used to control the stake pool. Three personas are supported: 
+- Owner
+- Operator
+- Voter
+
+Using this owner-operator-voter model, a custodian can assume the owner persona and stake on the Aptos blockchain and participate in the Aptos governance. This model allows delegations and staking services to be built as the owner can provide funds to the validator and the voter personas.
```

Comment:
It's not that "the owner can provide funds to the validator and the voter personas" but this separates control of funds from other accounts (operator, voter) and allows secure delegations of responsibilities.